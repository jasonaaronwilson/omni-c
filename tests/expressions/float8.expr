1.2E+1f;