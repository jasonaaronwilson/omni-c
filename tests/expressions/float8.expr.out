{
    tag = PARSE_NODE_LITERAL
    tokens = [
        1.2E+1f
    ]
}

// C Output
1.2E+1f
